ledger/src/blockstore_processor.rs

```diff
@@ -154,6 +154,20 @@ fn aggregate_total_execution_units(execute_timings: &ExecuteTimings) -> u64 {
     execute_cost_units
 }
 
+fn sum_additional_costs(batch: &TransactionBatch) -> u64 {
+    let mut additional_costs: u64 = 0;
+    let transactions = batch.sanitized_transactions();
+    let cost_model = CostModel::new();
+
+    for transaction in transactions {
+        let transaction_cost = cost_model.calculate_cost(transaction);
```

Comment:
+1 for using exact same method to compute tx cost, tho not sure if there's performance impact. 

On the same token, maybe looking into possibility of replacing `cost_capacity_meter` with `cost_tracker` to avoid duplicated logic?

Reply:
I like cost capacity meter for its simplicity, I think. Using the CostTracker to handle this is a large hammer for a very small nail, and I think it would be more confusing (because it tracks all sorts of stuff, reports it, etc., none of which we need here).

Reply:
The motivation of this change is to have a safety check to prevent validator being sucked into replaying a large block for long time. So a "large block" could be one that exceeds block_max_limit (capacity meter sort of cover that), or could be one has txs that exceeds account_max_limit, which is not checked. Cost_tracker handles both case, would be good to reuse it for consistency.